### ISSUE TITLE
Pressing tab after a bullet list should insert a new bullet item

### ISSUE THREAD

**Issue by provokateurin** [nextcloud/text#6248]
**Describe the bug**
Using the tab key in a bullet list indents items, so I would expect the tab key to insert a new bullet item, if I'm after the list but not inside it set.
I checked LibreOffice Writer and it doesn't do this, but the current behavior is also quite annoying because it moves you to somewhere else in the document.

**To Reproduce**
1. Create a bullet list
2. Place the cursor after the list
3. Press tab

**Expected behavior**
A new bullet item should be added.
Instead the tab will focus the next element, e.g. an image description or something else.
To add a new bullet item one has to press backspace instead which is not so convenient.

**Client details:**
 - OS: Arch Linux
 - Browser: Firefox
 - Browser version: 129
 - Device: Desktop

**Comment by juliushaertl** [comment]
I'm not sure I would expect this kind of behavior, if not in a list we currently have the tab key to focus to the next focus able element in tab order.

Do you have any examples of other editors behaving like that?

**Comment by provokateurin** [comment]
Not really. Maybe this is not standard behavior. But the current behavior is also very annoying because it jumps somewhere completely different in the document which often breaks my flow.
In my case I am writing a bullet list, focus another window and then focus the editor again. The cursor will be placed after the list and then I can not use tab to create a new item.
Maybe this is a focus issue instead because the cursor shouldn't be placed after the list but at the end of the last item? I didn't insert a newline after the list, so this is not really what I expected.

**Comment by mejo-** [comment]
I agree with Julius that this would be unexpected behaviour. We only allow indenting list items via tab shortcut when the cursor is inside a list item.

The bug with cursor jumping after the list when switching windows is no longer reproducible for me.
